zaza/openstack/charm_tests/charm_upgrade/tests.py

```diff
@@ -22,9 +22,9 @@
 import zaza.model
 from zaza.openstack.utilities import (
     cli as cli_utils,
+    os_versions as os_versions,
     upgrade_utils as upgrade_utils,
```

Comment:
Wouldn't:

```
from zaza.openstack.utilities import (
    cli as cli_utils,
    os_versions,
    upgrade_utils,
)
```

also work and be simpler?

Reply:
Yes, definitely